### PR TITLE
[Plot] Set yKey on each plot series

### LINKED ISSUE #5785: Allow configuration of yKey for each series of a plot (Overlay)
**Is your feature request related to a problem? Please describe.**
Currently when a yKey is set for a plot, it is automatically saved to the first series of a plot (even if the plot is composed of multiple series)

**Describe the solution you'd like**
When the yKey is saved on a plot, save the yKey to all the series of the plot
https://github.com/nasa/openmct/blob/master/src/plugins/plot/MctPlot.vue#L1101
This will allow the user to later specify different yKeys for each series depending on the y-axis they are assigned to.

**Describe alternatives you've considered**
None

**Additional context**
This is the first step to getting multiple Y Axis into plots for Open MCT

### PR DESCRIPTION
**Pull request by khalidadil**
Closes #5785 

### Describe your changes:
Saving the yKey to each series of the plot

### All Submissions:

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [X] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [X] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [X] Command line build passes?
* [X] Has this been smoke tested?
* [X] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by ozyx** [comment]
Should this be set to merge into https://github.com/nasa/openmct/tree/feature/plots-multiple-y-axes ? @shefalijoshi

**Comment by khalidadil** [comment]
> Should this be set to merge into https://github.com/nasa/openmct/tree/feature/plots-multiple-y-axes ? @shefalijoshi

Good call, I think that's right. I just updated it!

File: src/plugins/plot/pluginSpec.js [hunk 1]
```diff
@@ -410,6 +410,20 @@ describe("the plugin", function () {
             expect(options[1].value).toBe("Another attribute");
         });
 
+        it("Updates the Y-axis label when changed", () => {
```
Comment (khalidadil) [inline src/plugins/plot/pluginSpec.js:413]: @shefalijoshi Let me know if this test makes sense!

Comment (shefalijoshi) [reply src/plugins/plot/pluginSpec.js:413]: This looks good. Thanks!

**Pull request merged**